framework/src/problems/DisplacedProblem.C

```diff
@@ -278,7 +281,11 @@ DisplacedProblem::updateMesh(bool mesh_changing)
   _mesh.getMesh().clear_point_locator();
 
   // The mesh has changed. Face information normals, areas, etc. must be re-calculated
-  _mesh.finiteVolumeInfoDirty();
+  if (haveFV())
+  {
+    _mesh.finiteVolumeInfoDirty();
+    _mesh.buildFiniteVolumeInfo();
```

Comment:
Usually this just got rebuilt as needed by other calls but I suppose it's fine to call it explicitly

Reply:
Yeah I am removing the calls from accessors.

Reply:
My reasoning is that I would not like the mix the functionalities. I extended the assert in the accessor when the lookup doesn't find anything.